src/include/dhandle.h

```diff
@@ -74,6 +74,7 @@ struct __wt_data_handle {
 	uint64_t name_hash;		/* Hash of name */
 	const char *checkpoint;		/* Checkpoint name (or NULL) */
 	const char **cfg;		/* Configuration information */
+	const char *meta_base;  /* Base metadata configuration */
```

Comment:
Nit: This should be tabbed out to align with the other comments.

Reply:
fixed